docs/user/howto/shared-deps.md

```diff
@@ -0,0 +1,165 @@
+# Working with shared dependencies
+
+Hypper charts understand the concept of shared dependency charts.
+
+A chart declared as a shared dependency is a chart that more than one chart may
+depend on; once deployed, it can be reused by multiple other deployments. Charts
+deployed as shared dependencies are the analogous of system libraries in an OS:
+dependencies that are used by several consumers.
+
+## Creating a Hypper chart with shared dependencies
+
+Let's create the simplest chart possible, and add charts as shared dependencies
+to it.
+
+First, we create a simple empty chart:
+
+```console
+$ helm create our-app
+```
+
+Now, we can edit `./our-app/Chart.yaml`, and add some shared dependencies to it:
+
+```diff
+apiVersion: v2
+name: our-app
+description: A Helm chart for Kubernetes
+type: application
+version: 0.1.0
+appVersion: 1.16.0
+annotations:
++ hypper.cattle.io/shared-dependencies: |
++   - name: postgresql
++     version: "~8.9.4"
++     repository: "https://charts.bitnami.com/bitnami"
++   - name: prometheus
++     version: "~13.7.0"
++     repository: "https://prometheus-community.github.io/helm-charts"
+```
+
+Shared dependencies are just normal Helm
+[Dependencies](https://helm.sh/docs/topics/charts/#chart-dependencies). As
+such, they are defined with:
+- The name field is the name of the chart you want.
+- The version field is the version of the chart you want.
+- The repository field is the full URL to the chart repository. Note that you
+  must also use helm repo add to add that repo locally. You might use the name
```

Comment:
hypper instead of helm, otherwise the tutorial wont work :p

Reply:
woooops! fixed in https://github.com/rancher-sandbox/hypper/pull/131/commits/50f8d779c7984678c6ab6e82b4e60a176a26c51c.